src/mono/mono/mini/interp/jiterpreter.c

```diff
@@ -437,6 +438,43 @@ mono_jiterp_conv_ovf (void *dest, void *src, int opcode) {
 	return 0;
 }
 
+#define JITERP_RELOP(opcode, type, op, noorder) \
+	case opcode: \
+		{ \
+			if (is_unordered) \
+				return noorder; \
+			else \
+				return ((type)lhs op (type)rhs); \
+		}
+
+EMSCRIPTEN_KEEPALIVE int
+mono_jiterp_relop_fp (double lhs, double rhs, int opcode) {
+	gboolean is_unordered = mono_isunordered (lhs, rhs);
+	switch (opcode) {
+		JITERP_RELOP(MINT_CEQ_R4, float, ==, 0);
```

Comment:
seems cleaner to pass `TRUE`/`FALSE`

Reply:
This is copied from interp.c, but I can change it

Reply:
Oh, nevermind. It seemed to me that this argument was used during a conditional.